docs/requirements.md

```diff
@@ -2,78 +2,48 @@
 
 Running Codacy on a Kubernetes cluster requires the following:
 
--   A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#cluster-resource-requirements)
--   The [NGINX Ingress Controller](https://github.com/helm/charts/tree/master/stable/nginx-ingress) for Kubernetes
+-   A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#hardware-requirements)
+-   The [NGINX Ingress Controller](https://github.com/helm/charts/tree/master/stable/nginx-ingress) must be setup in your cluster
 -   A [PostgreSQL server](#postgresql-server-setup) accessible from the Kubernetes cluster
 
-## Cluster resource requirements
+## Hardware Requirements
 
-To have an enjoyable experience with Codacy, you should have the
-following calculations in mind when allocating resources for the
-installation and defining the number of concurrent analysis.
+A good way of understanding how resources are used in Codacy is by understanding how the resources are allocated.
 
-Without accounting for analysis ([next section](#analysis)),
-you should need at least the following resources:
+!["High Level Architecture"](<./images/High Level Architecture - Analysis II.svg> "High Level Architecture")
```

Comment:
I think that later we can simplify the diagram a bit - or at least highlight better what's relevant for this page.

Ideally, we should have the architecture of the platform described on the documentation so that here we could simply refer to that.

Reply:
Can you help me edit it?
https://docs.google.com/drawings/d/1n4FxUB1UeHMxSKN28U1zD1FmzWJ6Ewq3p3sJbXwNoSM/edit?usp=drive_web&ouid=116146841954292499642